connectors/v2/pakartot.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#player';
+
+Connector.artistSelector = 'div.jp-player-artist > a';
+
+Connector.trackSelector = 'div.jp-player-title > a';
+
+Connector.playButtonSelector = '.jp-play';
+
+Connector.getArtist = function() {
+	var text = $(this.artistSelector).text().trim();
```

Comment:
> Or move this logic: https://github.com/david-sabata/web-scrobbler/blob/master/connectors/v2/bandcamp.js#L17 to core? something like some base connector cleanup?

I think it's should be in the core if it's used by more than one connector.

What about implementation. I think that's all stuff should be placed into some `util` module, not in the `connector` module. Youtube shared metadata filter should be placed into that module too. Then, these all functions can be easily covered by unittests (the unittest core is almost done).